sw-emulator/lib/periph/src/soc_reg.rs

```diff
@@ -679,6 +681,20 @@ impl SocRegistersImpl {
         }
     }
 
+    fn set_cptra_security_state_device_lifecycle(&mut self, args: &CaliptraRootBusArgs) {
+        let mut value = SecurityState::LIFE_CYCLE::UNKNOWN;
```

Comment:
Can we make unprovisioned default instead of unknown

Reply:
Updated